pkg/subctl/cmd/gather.go

```diff
@@ -203,22 +198,39 @@ func gatherDiscovery(dataType string, info *gather.Info) bool {
 	return true
 }
 
-func gatherBroker(dataType string, info *gather.Info) bool {
+func gatherBroker(dataType string, info gather.Info) bool {
 	switch dataType {
 	case Logs:
-		info.Status.QueueWarningMessage("No logs to gather on Broker")
+		info.Status.QueueSuccessMessage("No pods accessible. No logs to collect on Broker")
```

Comment:
This isn't exactly true - we do deploy an operator pod on the broker. I think we should just remove these 2 lines so nothing is outputted - we can add it back if we decide to gather the pod logs.